setup.py

```diff
@@ -1864,9 +1864,10 @@ def detect_modules(self):
 ##         # Uncomment these lines if you want to play with xxmodule.c
 ##         self.add(Extension('xx', ['xxmodule.c']))
 
-        # Limited C API
-        self.add(Extension('xxlimited', ['xxlimited.c']))
-        self.add(Extension('xxlimited_35', ['xxlimited_35.c']))
+        # Build limited C API without Py_TRACE_REFS macro
```

Comment:
This comment is wrong. You should say that the limited C API is not compatible with the Py_TRACE_REFS macro.

Reply:
Got it. Updated :)